Cargo.toml

```diff
@@ -231,6 +231,10 @@ unused_peekable = "warn"
 # Diagnostics are not actionable: Enable once https://github.com/rust-lang/rust-clippy/issues/13774 is resolved.
 large_stack_arrays = "allow"
 
+# Salsa generates functions with parameters for each field of a `salsa::interned` struct.
+# If we don't allow this, we get warnings for structs with too many fields.
+too_many_arguments = "allow"
```

Comment:
I didn't find a way to only set this for `red_knot_python_semantic`, as it inherits this configuration, and it looks like I can't overwrite workspace settings for lints?

Reply:
I'll merge this for now, but I will open a salsa ticket to improve this, and add a reminder for myself to remove this again.